Gems/PhysX/Code/Tests/Benchmarks/PhysXRigidBodyBenchmarks.cpp

```diff
@@ -372,6 +457,98 @@ namespace PhysX::Benchmarks
         SetLabel(state, bodyType);
     }
 
+    //! BM_RigidBody_Activation - This test will create the requested number of rigid bodies, including
+    //! mock components that depend on the rigid bodies, and measure the time it takes to activate them.
+    BENCHMARK_DEFINE_F(PhysXRigidbodyBenchmarkFixture, BM_RigidBody_Activation)(benchmark::State& state)
+    {
+        // get the request number of rigid bodies and prepare to spawn them
+        const int numRigidBodies = aznumeric_cast<int>(state.range(0));
+
+        const float boxSize = 1.0f;
+        const float boxSpacing = 0.25f;
+
+        // common settings for each rigid body
+        const float boxSizeWithSpacing = boxSize + boxSpacing;
+        const int boxesPerCol = static_cast<const int>(RigidBodyConstants::TerrainSize / boxSizeWithSpacing) - 1;
+        int spawnColIdx = 0;
+        int spawnRowIdx = 0;
```

Comment:
nit/standards: Prefer to avoid abbreviations and just use `Index` or `index` instead

Reply:
Will change.

Reply:
Awesome thanks 👍